src/main/resources/static/template/js/fillRoomData.js

```diff
@@ -43,7 +47,7 @@ function viewAllRoomsData(startDate, endDate, amountOfGuests){
                             value.roomType.singleBeds + ' single and '+ value.roomType.doubleBeds + " double beds.</h4></a>\
                             <h6>{price}<small>/night</small></h6>\
                             </div>\
-                         </div>";
+                            </div>";
 
```

Comment:
waarom html in javascript? Je kan ook in javascript een element in een div stoppen met een id. En div hiden via javascript.